src/Sylius/Bundle/PromotionBundle/Resources/config/services.xml

```diff
@@ -80,6 +80,6 @@
             <argument type="service" id="sylius.repository.catalog_promotion" />
         </service>
 
-        <service id="Sylius\Component\Promotion\Provider\DateTimeProviderInterface" class="Sylius\Component\Promotion\Provider\Calendar" />
+        <service id="Sylius\Calendar\Provider\DateTimeProviderInterface" class="Sylius\Behat\Service\Provider\Calendar" />
```

Comment:
this service should be replaced by `Sylius\Calendar\Provider\CalendarInterface` from `sylius/calendar`